samples/SampleStartups/StartupHelloWorld.cs

```diff
@@ -6,26 +6,58 @@
 
 namespace SampleStartups
 {
-    public class StartupHelloWorld : StartupBase
+    public class Program
     {
-        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
-        public override void Configure(IApplicationBuilder app)
+        public static void Main()
         {
-            app.Run(async (context) =>
-            {
-                await context.Response.WriteAsync("Hello World!");
-            });
+            var host = new WebHostBuilder()
+                .Run(async context =>
+                {
+                    await context.Response.WriteAsync("Hello World");
+                });
+
+            host.WaitForShutdown();
+        }
+
+        public static void MainWithPort()
+        {
+            var host = new WebHostBuilder()
+                .Run(8080, async context =>
```

Comment:
url instead of port is probably ok. UseUrls is a little bit more but having Run(...) or some similar method just take the handler instead of calling Configure is a great simplification.